website/public/js/controllers/rootCtrl.js

```diff
@@ -21,7 +21,8 @@ habitrpg.controller("RootCtrl", ['$scope', '$rootScope', '$location', 'User', '$
         if (!!fromState.name) Analytics.track({'hitType':'pageview','eventCategory':'navigation','eventAction':'navigate','page':'/#/'+toState.name});
         // clear inbox when entering or exiting inbox tab
         if (fromState.name=='options.social.inbox' || toState.name=='options.social.inbox') {
-          User.user.ops.update && User.set({'inbox.newMessages':0});
+          //@TODO: Protected path. We need a url
+          User.set({'inbox.newMessages': 0});
```

Comment:
for this a route might be the best choice. Otherwise we can unprotect inbox.newMessages